worker/src/main/java/gov/cms/ab2d/worker/processor/coverage/CoverageProcessorImpl.java

```diff
@@ -1,17 +1,10 @@
 package gov.cms.ab2d.worker.processor.coverage;
 
 import gov.cms.ab2d.bfd.client.BFDClient;
-import gov.cms.ab2d.common.model.CoveragePeriod;
+import gov.cms.ab2d.common.repository.ContractRepository;
```

Comment:
Why is this back in use? Should we wrap it with the contract service instead so that the service can be rewritten as a REST api call in the future?

Reply:
Because In this code I need the contract type in contract table. In coveragemapping I only get contractNumber. The type is a required to check if the contract is a testing one.

Reply:
Why not pass the ContractForCoverageDTO instead?

Reply:
Because then the question is where the contract data comes from where coveragemapping is used. ContractService. I did switch ContractRepository with contract service. But no matter what we will need to grab the contract data in some way. We could make contractDTO use javax. But then we might as well keep coverage using the common contract class.